kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -63,17 +66,21 @@ public S3SinkTask() {
     // no-arg constructor required by Connect framework.
     assignment = new HashSet<>();
     topicPartitionWriters = new HashMap<>();
+    time = new SystemTime();
```

Comment:
Oh, I guess I never even noticed this. Is there some place we assume the `io.confluent` version? I thought we had kept them pretty close to each other, with the Kafka one more recently getting a minor upgrade re: hi res clocks.